### PR TITLE
integrate cosmos kit for ibc

### PR DESCRIPTION
**Pull request by turbocrime**
todo

- [ ] style #848 #850 #852
- [ ] restore outbound tx from early version
- [ ] outbound denom display
- [ ] construct inbound tx
- [ ] properly filter 'out' assets to destination chain
- [ ] display 'in' assets in-page, or in wallet dialog?
- [ ] fetch cosmos wallet balances (getting 405/429 errors. debounce?)
- [ ] move 'chain select' to be a page level instead of form level component

### CODE REVIEW

File: apps/minifront/src/components/ibc/layout.tsx [hunk 1]
```diff
@@ -2,15 +2,46 @@ import { Card } from '@penumbra-zone/ui/components/ui/card';
...
+import { ibcSelector } from '../../state/ibc';
+import { useLoaderData } from 'react-router-dom';
+import { useEffect } from 'react';
+import { Chain as PenumbraChain } from '@penumbra-zone/constants/src/chains';
+
 export const IbcLayout = () => {
+  useLoaderData();
+  const { penumbraChain } = useStore(ibcSelector);
+
+  console.log('## ibc-layout', penumbraChain?.chainName);
```
Comment (turbocrime) [inline apps/minifront/src/components/ibc/layout.tsx:17]: useChain refuses to conditionally execute or execute with an empty chainName, but the way we've set up zustand forces a refresh with 'undefined' when changing the chainName state.

**Comment by grod220** [comment]
Think this PR is missing descriptions, screenshots (or video walkthrough), and code comments (or github comments) explaining the code more. After that, feel free to re-request review.

File: packages/constants/package.json [hunk 2]
```diff
@@ -16,5 +16,8 @@
     "clean": "rm -rfv dist",
     "lint": "eslint \"**/*.ts*\"",
     "test": "vitest run"
+  },
+  "dependencies": {
+    "chain-registry": "^1.33.25"
```
Comment (turbocrime) [inline packages/constants/package.json:21]: this might bloat the export

**Comment by turbocrime** [comment]
closed in favor of smaller scope #948

**Pull request closed without merging**